governance/votes/Executive Vote - August 2, 2023.md

```diff
@@ -0,0 +1,141 @@
+
+| AVC Member Name  | Address                                                                                                                | AVC                 | Amount (MKR) |
+|------------------|------------------------------------------------------------------------------------------------------------------------|---------------------|--------------|
+| IamMeeoh         | [0x47f7A5d8D27f259582097E1eE59a07a816982AE9](]https://etherscan.io/address/0x47f7A5d8D27f259582097E1eE59a07a816982AE9) | KISS                | 14.9         |
+| ACRE DAOs        | [0xBF9226345F601150F64Ea4fEaAE7E40530763cbd](https://etherscan.io/address/0xBF9226345F601150F64Ea4fEaAE7E40530763cbd)  | Regnerative Finance | 14.9         |
+| Space Xponential | [0xFF8eEB643C5bfDf6A925f2a5F9aDC9198AF07b78](https://etherscan.io/address/0xFF8eEB643C5bfDf6A925f2a5F9aDC9198AF07b78)  | Regnerative Finance | 11.92        |
+| Res              | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784)  | Resiliency          | 14.9         |
+| LDF              | [0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50](https://etherscan.io/address/0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50)  | Composable          | 11.92        |
+| opensky          | [0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04](https://etherscan.io/address/0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04)  | Composable          | 14.9         |
+| David Phelps     | [0xd56e3E325133EFEd6B1687C88571b8a91e517ab0](https://etherscan.io/address/0xd56e3E325133EFEd6B1687C88571b8a91e517ab0)  | Composable          | 8.94         |
+| seedlatam.eth    | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d)  | Sovereign Finance   | 11.92        |
+| StableLab        | [0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08](https://etherscan.io/address/0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08)  | Growth              | 14.9         |
+| flipsidegov      | [0x300901243d6CB2E74c10f8aB4cc89a39cC222a29](https://etherscan.io/address/0x300901243d6cb2e74c10f8ab4cc89a39cc222a29)  | Growth              | 14.9         |
+
+This comes to a total of **134.1 MKR**.
+
+### DAO Resolution
+
+As per this forum [post](https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291) that has been approved by the Stability Scope Facilitators, the following DAO Resolution will be approved if this executive proposal passes.
+
+- Approve the DAO Resolution with the IPFS hash `QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p` - the DAO Resolution may be viewed [here](https://gateway.pinata.cloud/ipfs/QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p).
+
+In summary, this will instruct the trustees of the Monetalis Clydesdale (RWA007-A) JAT1 arrangement to sell a portion of ETF holdings and return profits to MakerDAO via the input conduit.
```

Comment:
```suggestion
In summary, this will instruct the trustees of the Monetalis Clydesdale (RWA007-A) JAT1 arrangement to sell a portion of ETF holdings and return the profits to the MakerDAO surplus buffer via the input conduit smart contract.
```